core/web/resolver/mutation.go

```diff
@@ -260,3 +262,34 @@ func (r *Resolver) UpdateFeedsManager(ctx context.Context, args struct {
 
 	return NewUpdateFeedsManagerPayload(mgr, nil, nil), nil
 }
+
+func (r *Resolver) CreateOCRKeyBundle(ctx context.Context) (*CreateOCRKeyBundlePayloadResolver, error) {
+	if err := authenticateUser(ctx); err != nil {
+		return nil, err
+	}
+
+	key, err := r.App.GetKeyStore().OCR().Create()
+	if err != nil {
+		return nil, err
+	}
+
+	return NewCreateOCRKeyBundlePayloadResolver(key), nil
+}
+
+func (r *Resolver) DeleteOCRKeyBundle(ctx context.Context, args struct {
+	ID string
+}) (*DeleteOCRKeyBundlePayloadResolver, error) {
+	if err := authenticateUser(ctx); err != nil {
+		return nil, err
+	}
+
+	deletedKey, err := r.App.GetKeyStore().OCR().Delete(args.ID)
+	if err != nil {
+		if errors.As(err, &keystore.KeyNotFoundError{}) {
+			return NewDeleteOCRKeyBundlePayloadResolver(ocrkey.KeyV2{}, err), nil
+		}
```

Comment:
Let me know your thoughts on this as opposed to e.g `strings.HasPrefix` on `err.Error()`.

Reply:
This is much better